Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -28,14 +28,17 @@
 
 #if ENABLE(PDF_PLUGIN)
 
+#import "MessageSenderInlines.h"
 #import "PluginView.h"
 #import "WebEventConversion.h"
 #import "WebFrame.h"
 #import "WebPage.h"
+#import "WebPageProxyMessages.h"
 #import <CoreFoundation/CoreFoundation.h>
 #import <WebCore/AXObjectCache.h>
 #import <WebCore/ArchiveResource.h>
 #import <WebCore/Chrome.h>
+#import <WebCore/Cursor.h>
```

Comment:
This import is redundant now, but if you make the suggested change in https://github.com/WebKit/WebKit/pull/20561, it should be kept.

Reply:
Yeah that's a good point. I'll make it `WebCore::CursorType` and not pull in the `WebCore/Chrome.h` header in `PDFPluginBase.h`.